rpc/cloudagent/network/loadbalancer/moc_cloudagent_loadbalancer.proto

```diff
@@ -59,6 +89,11 @@ message LoadBalancer {
 	Tags tags = 11;
 	uint32 replicationCount = 12;
 	repeated LoadbalancerInboundNatRule inboundNatRules = 13;
+	repeated LoadbalancerOutboundNatRule outboundNatRules = 14;
```

Comment:
@kybeak Can you add the azure SDK api links here as a comment. This would be easier for anyonce adding new params in the future. Lets add the same for all the new changes

Reply:
Sure, I can add compatible Azure SDK link. For example, 
// ------------------------------------------------------------------------------
// Azure public IP address overview
// https://learn.microsoft.com/en-us/azure/virtual-network/ip-services/virtual-network-public-ip-address
// Azure CLI:
// https://learn.microsoft.com/en-us/cli/azure/network/public-ip?view=azure-cli-latest
// ------------------------------------------------------------------------------